RELEASE/scripts/sl_ascend/sl_util.ash

```diff
@@ -5032,17 +5049,17 @@ boolean sl_is_valid(item it)
 		else if(!expectGhostReport() && !haveGhostReport())
 			return false;
 	}
-	return is_unrestricted(it);
+	return bees_hate_usable(it.to_string()) && is_unrestricted(it);
```

Comment:
I believe some items are still valid in bees hate you despite having Bs, like the beehive (ironically). Also you CAN equip items with Bs in bees hate you, iirc, they just cause you to take passive damage, or something like that?